openbase/planetsudo.engine
# 27 introduce new resource type called tonic

### :scroll: Description

Changes proposed in this pull request:
* Introduce Tonic Resource

### 🧪 Test Instructions

* Test with the "Kreuzung"
* Make sure Agents are able to collect at most 3 tonics.
* Make sure the white / blue bar above the agent visualizes the tonic level that has been collected.

### :white_check_mark: Checklist:

- [x] Extended the documentation

### :camera_flash: Screenshots

![Screenshot 2024-02-11 at 17 17 00](https://github.com/openbase/planetsudo.engine/assets/6483977/b6921441-55f5-4375-b8bc-73f9806bf097)
![Screenshot 2024-02-11 at 17 17 59](https://github.com/openbase/planetsudo.engine/assets/6483977/8655efc3-0626-4db0-9dd1-a71f902ebd51)
![Screenshot 2024-02-11 at 17 17 16](https://github.com/openbase/planetsudo.engine/assets/6483977/304ca8f0-b519-402e-8f95-129889487f04)
